gammapy/makers/tests/test_utils.py

```diff
@@ -332,6 +350,44 @@ def test_make_edisp_kernel_map():
     assert_allclose(kernel.pdf_matrix[:, 2], (0.0, 0.0, 0.0, 0.0, 1.0, 1.0), atol=1e-14)
 
 
+@requires_data()
+def test_make_counts_rad_max(observations):
```

Comment:
But this is already tested no?

Reply:
If it is already tested I didn't find the test.